config.properties.template

```diff
@@ -77,6 +78,9 @@ sort_by_iv=false
 # Always throw curveballs
 always_curve=false
 
+# Never use berries
+never_use_berries=true
```

Comment:
Did this will blocked all berry using? My suggestion is berries not using with redball together, change to blue ball first, if probability if also low, try berry then.